cmd/autogen.sh

```diff
@@ -38,9 +38,16 @@ case "$ID" in
 				extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --enable-static-libcap --enable-static-libapparmor --enable-static-libseccomp"
 				;;
 			*)
-				extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --enable-static-libcap"
+				extra_opts="--libexecdir=/usr/lib/snapd --enable-static-libcap --enable-nvidia-multiarch"
```

Comment:
Can we drop the variant altogether now? This code should handle old and new model.

Reply:
`--enable-static-libcap --enable-static-libapparmor --enable-static-libseccomp` for all then?